inst/json/graph_prevalence_config.json

```diff
@@ -31,46 +35,34 @@
             "id": "llin",
             "name": "Pyrethroid LLIN only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "blue"}
         },
         {
             "id": "irs",
             "name": "IRS only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "purple"}
         },
         {
             "id": "irs-llin",
             "name": "Pyrethroid LLIN with IRS",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "blue"}
         },
         {
             "id": "llin-pbo",
             "name": "Pyrethroid-PBO LLIN only",
             "type": "lines",
-            "marker": {"color": "grey"}
+            "marker": {"color": "aquamarine"}
         },
         {
             "id": "irs-llin-pbo",
             "name": "Pyrethroid-PBO LLIN with IRS",
             "type": "lines",
-            "marker": {"color": "grey"}
-        },
-        {
-             "x": [12, 12],
-             "y": [0, 1],
-             "line": {
-                 "dash": "dot",
-                 "width": 2,
-                 "color": "grey"
-             },
-             "name": "",
-             "showlegend": false,
-             "hoverinfo": "none"
-         }
+            "marker": {"color": "aquamarine"}
```

Comment:
two series both with aquamarine?

Reply:
Yes, so I got a bit confused because I misremembered how the original app showed things. In that app you actually see all 6 series at once. However, the way we're now filtering the data by intervention options, you'll only get the series which are relevant eg you won't see 'irs only' or 'no intervention' if you've selected positive values for both irs and nets. Ellie's comment on the Teams thread on this suggests that she's happy just to see the relevant series, as long as we can optionally show no intervention alongside it, as showing all the series might be too confusing.

I had misremembered that we always showed standard net and pbo net series as blue and aqua on this graph, whether with irs or not, but they should actually be all separate colours. Will update to match the colours we use in the cost graphs. But I expect there will still be some usability tweaking.